postgres/rdspostgres/rdspostgres.go

```diff
@@ -53,6 +55,40 @@ type Params struct {
 	TraceOpts []ocsql.TraceOption
 }
 
+type URLOpener struct {
+	// TraceOpts contains options for OpenCensus.
+	TraceOpts []ocsql.TraceOption
+}
+
+// Scheme is the URL scheme rdspostgres registers its URLOpener under on
+// postgres.DefaultMux.
+const Scheme = "rdspostgres"
+
+func init() {
+	postgres.DefaultURLMux().RegisterPostgres(Scheme, &URLOpener{})
+}
+
+func (uo *URLOpener) OpenPostgresURL(ctx context.Context, u *url.URL) (*sql.DB, error) {
+	cf := new(rds.CertFetcher)
+	for k, _ := range u.Query() {
+		// Only permit parameters that do not conflict with other behavior.
+		if k == "user" || k == "password" || k == "dbname" || k == "host" || k == "port" || k == "sslmode" || k == "sslcert" || k == "sslkey" || k == "sslrootcert" {
+			return nil, fmt.Errorf("cloudpostgres: OpenPostgresURL: extra parameter %s not allowed; use Params fields instead", k)
+		}
+	}
+	vals := u.Query()
+	vals.Set("sslmode", "disable")
```

Comment:
By and the large, the utility of this package is to connect to RDS databases securely using TLS. We shouldn't be disabling this (see with comment below).